std/equal.m31

```diff
@@ -427,7 +427,7 @@ let rec equality betas hints etas = handler
 
 (* TODO | as_* *)
 
-  | val ?v => fun betas hints etas => pair v (betas :: hints :: etas :: [])
+  | val ?v => fun betas hints etas => (v, betas :: hints :: etas :: [])
```

Comment:
This should probably be converted to a tuple `(v, (betas,hints,etas))`, as well as in the patterns using it.